code/modules/admin/verbs/superheroes.dm

```diff
@@ -0,0 +1,58 @@
+
+/client/proc/summon_superheroes()
+	set category = "Admin.Fun"
+	set name = "Summon Superheroes and Supervillains"
+	set desc = "Spawn superhero and supervillain ships. USE ONLY ONCE OR SHIPS MAY BREAK."
+
+	var/choice = input(src, "Which teams do you want to spawn?") in list("Superheroes", "Supervillains", "Both superheroes and supervillains")
+
+	if(choice == "Superheroes" || choice == "Both superheroes and supervillains")
+		holder?.spawnSuperheroes()
+
+	if(choice == "Supervillains" || choice == "Both superheroes and supervillains")
+		holder?.spawnSupervillains()
+
+	var/list/candidates = pollGhostCandidates("Would you like to play as a superhero/supervillain?", ROLE_SUPERHERO, FALSE, 300)
+	var/list/heroes = list()
+	for(var/obj/effect/mob_spawn/human/superhero/spawner in world)
+		heroes.Add(spawner)
+
+	while(LAZYLEN(candidates) && LAZYLEN(heroes))
+		var/mob/dead/observer/ghostie = pick_n_take(candidates)
+		var/obj/effect/mob_spawn/human/superhero/spawner = pick_n_take(heroes)
+		spawner.create(ckey = ghostie.ckey)
+
+	message_admins("[key_name(usr)] spawned [choice].")
+	log_admin("[key_name(usr)] spawned [choice].")
+
+	if(choice == "Both superheroes and supervillains")
+		priority_announce("Two unidentified ships detected near the station.")
```

Comment:
will the ships still spawn if no ghosts say yes to their polls?

Reply:
Yes and that's intended because ghosts can spawn using cryopods